src/app.js

```diff
@@ -131,6 +130,7 @@ export default function createApp(rootElement) {
   const createTextEl = (text) => {
     const p = document.createElement("p");
     p.classList.add("todo-text");
+    p.setAttribute("data-testid", "text-element");
```

Comment:
the "text element" of a todo item should be found by its content e.g. "Call doctor"